abstract_specification.md

```diff
@@ -49,78 +45,78 @@ class DataFrame():
+* `SOMAExperiment`: a specialization and extension of `SOMACollection`, codifying a set of naming and indexing conventions to represent an annotated, 2-D matrix of observations across _multiple_ sets of variables.
 
 In this document, the term `dataframe` implies something akin to an Arrow `Table` (or `RecordBatch`), R `data.frame` or Python `pandas.DataFrame`, where:
 
-- multiple columns may exist, each with a string column name
-- all columns are individually typed and contain simple data types (e.g., int64)
-- all columns are of equal length
-- rows are addressed by one or more dataframe columns
+* multiple columns may exist, each with a string column name;
+* all columns are individually typed and contain simple data types (e.g. `int64`);
+* all columns are of equal length; and
+* rows are addressed by one or more dataframe columns.
 
 All SOMA data objects are named with URIs.
 
 ## Base Type System
 
-The SOMA API borrows its base type system from the Arrow language-agnostic in-memory system for data typing and serialization ([format](https://arrow.apache.org/docs/format/Columnar.html)). The SOMA API is intended to be used with an Arrow implementation such as [PyArrow](https://arrow.apache.org/docs/python/) or the [Arrow R package](https://arrow.apache.org/docs/r/), or other libraries which interoperate with Arrow (e.g., Pandas).
+The SOMA API borrows its base type system from the Arrow language-agnostic in-memory system for data typing and serialization ([format](https://arrow.apache.org/docs/format/Columnar.html)). The SOMA API is intended to be used with an Arrow implementation such as [PyArrow](https://arrow.apache.org/docs/python/) or the [Arrow R package](https://arrow.apache.org/docs/r/), or other libraries which interoperate with Arrow (e.g. Pandas).
 
-Where SOMA requires an explicit typing system, it utilizes the Arrow types and schema. SOMA has no specific requirements on the type or serialization system used by the underlying storage engine, other than it be capable of understanding and representing the Arrow types. It is expected that any given implementation of SOMA will have limits on the underlying capabilities of its data type system (e.g., just because you can express a type in the Arrow type system does not mean all SOMA implementations will understand it).
+Where SOMA requires an explicit typing system, it utilizes the Arrow types and schema. SOMA has no specific requirements on the type or serialization system used by the underlying storage engine, other than it be capable of understanding and representing the Arrow types. It is expected that any given implementation of SOMA will have limits on the underlying capabilities of its data-type system (e.g. just because you can express a type in the Arrow type system does not mean all SOMA implementations will understand it).
```

Comment:
according to much google searching on my part (lolz), in American English, a comma always follows 'e.g.'